Lib/test/support/__init__.py

```diff
@@ -856,6 +856,14 @@ def check_cflags_pgo():
     return any(option in cflags_nodist for option in pgo_options)
 
 
+def check_bolt_optimized():
+    # Always return false, if the platform is WASI.
```

Comment:
This comment only says _what_ the code does, but not _why_. Could you rewrite it to state why this is happening? I can easily see by the code that we always return false if we're on WASI; there is no need to manifest that info into a comment ;)